app/views/stars/_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for(@star) do |f| %>
+  <%= f.input :name, :required => false %>
+  <%= f.input :price, :required => false %>
+  <hr>
```

Comment:
what is `<hr>` ?

Reply:
c'est la fine ligne qui sépare le form du bouton

Reply:
ok